scss/civicrm/common/_notifications.scss

```diff
@@ -99,9 +99,31 @@
   }
 }
 
-.crm-container > .status,
-.crm-container :not(.crm-footer) .status {
-  padding: 15px;
+.crm-container {
+  > .status,
+  :not(.crm-footer) .status {
+    background-color: $alert-warning-bg;
+    border-color: $brand-warning;
+    color: $gray-darker;
+    padding: 16px 20px;
+
+    p {
+      &:first-child {
+        margin-top: 0;
+      }
+
+      &:last-child {
+        margin-bottom: 0;
+      }
+    }
+  }
+
+  .crm-block,
+  form {
```

Comment:
`form` is needed for status messages that are directly inside forms, for example when deleting contributions.